pages/about.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-GB">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+  </head>
+      <body>
+        <div class="parent">
+            <div class="div1">   
+                <nav class="navigationbar" >
+                <a href="#Squirell" id ="logo"> ᵋ₍◕ᴥٛ◕₎ᵌ </a> 
+                <a href="../pages/playground.html" id ="icanbranch">I can branch!</a>
+                <a href="../pages/index.html">Home</a>
+                <a href="../pages/about.html">About</a>
+                <a href="../pages/portfolio.html">Portfolio</a></nav>
+            
+            <div class="div2"></div>
+            <div class="div3"></div>
+            <div class="div4"></div>
+            <div class="div5"></div></div></div>
+          
+        <div class="parent2">
+            <div class= section-title-container > 
+              <h1 id="large-title">ABOUT </h1>
+            
+        </div>
+        <div class= body-padding-no-border style="background-color: white;"> 
```

Comment:
You should avoid using inline styles.  Pages are much easier to maintain when all of your styles live in stylesheets.